shared/components/StudyInfo/Details.tsx

```diff
@@ -102,6 +102,30 @@ const CollapseIcon = ({
   </>
 )
 
+const ExpandIcon = ({
+  showExpand,
+  toggle,
+  intl
+}: {
+  showExpand: boolean
+  toggle: (name: ToggleableItems) => () => void
+  intl: InjectedIntl
+}) => (
+  <>
+    {showExpand && (
```

Comment:
Meybe not needed boolean

Reply:
I think it is used in the case that there isn't any osaamisenOsoittamiset and osaamisenHankkimistavat. There isn't any extra data to show and that's why expand icon is not shown.